MAUI/PDF-Viewer/Coordinate-Conversion.md

```diff
@@ -0,0 +1,119 @@
+{
+    //Retrieve the tapped client area position.
+    Point clientPoint = e.Position;
+    //Retrieve the page number that corresponds to the client point.
+    int pageNumber = pdfViewerControl.GetPageNumberFromClientPoint(clientPoint);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## PDF page coordinates
+
+PDF page coordinates are represented in terms of device-independent coordinate system called user space, which is independent of the output device that will be used for printing or display. The user space coordinate system is initialized to a default state for each page of a PDF document. The length of a unit is 1/72 inch, which is approximately the same as a unit of point(pt). As an example, the dimensions of a letter-sized paper in PDF page coordinates are shown in the diagram below.
+
+![Coordinate conversion in .NET MAUI PDF Viewer](Images\Coordinate-Conversion\Page-Coordinates.jpg)
+
+## Convert PDF Viewer’s coordinates to PDF page coordinates
+
+You can obtain the PDF page coordinates using the `ConvertClientPointToPagePoint(clientPoint, pageNumber)` method by passing the client rectangle point and page number as input parameters. The following code sample explains how to convert a tapped client area position to a page point.
+
+{% tabs %}
+{% highlight c# %}
+//Subscribe the Tapped event of PDF Viewer
+pdfViewerControl.Tapped += PdfViewerControl_Tapped; 
+ 
+// Handle the tapped event.
+private void PdfViewerControl_Tapped(object sender, GestureEventArgs e)
+{
+    //Retrieve the tapped client area position.
+    Point clientPoint = e.Position;
+    //Retrieve the page number that corresponds to the client point.
+    int pageNumber = pdfViewerControl.GetPageNumberFromClientPoint(clientPoint)
+    //Retrieve the page point.
+    Point pagePoint = pdfViewerControl.ConvertClientPointToPagePoint(clientPoint, pageNumber);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Convert PDF page coordinates to PDF Viewer’s coordinates
+
+You can obtain the PDF page coordinates using the `ConvertPagePointToClientPoint(pagePoint, pageNumber)` method by passing the page point and page number as input parameters. The following code sample explains how to convert a square annotation’s position in the page coordinates to a client point.
+
+{% tabs %}
+{% highlight c# %}
+//Get a shape annotation from the Annotations collection.
+SquareAnnotation squareAnnotation = pdfViewerControl.Annotations[0] as SquareAnnotation;
+//Get the annotation’s page number.
+int pageNumber = squareAnnotation.PageNumber;
+//Get the annotation bounds.
+Rectangle annotationBounds = squareAnnotation.Bounds;
+//Find the position of the annotation in page coordinates.
+Point pagePoint = new Point(annotationBounds.X, annotationBounds.Y);
+//Convert the page point to client point.
+Point clientPoint = pdfViewerControl.ConvertPagePointToClientPoint(pagePoint, pageNumber);
+{% endhighlight %}
+{% endtabs %}
+
+## Convert PDF page coordinates to scroll coordinates
+
```

Comment:
Explain what do you mean by scroll coordinates and its unit. Its difference from client coordinates and its origin kind of details

Reply:
addressed